tests/test_api_template.py

```diff
@@ -71,10 +75,10 @@ def setup_template_dir(config_dict):
         print(TEST_JSON_PATH)
         file_path = os.path.join(TEMPLATE_DIR,"{}.json".format(TEMPLATE_UUID))
         shutil.copy(TEST_JSON_PATH,file_path )
-        new_modified_date = datetime.now() - timedelta(hours=24)
+        new_modified_date = datetime.now(tz.utc) - timedelta(hours=24)
```

Comment:
@vedina Please check if UTC time makes sense here. For some reason the original code used `now()`, not `utcnow()`, but it seems to me this might've simply been an oversight.